requirements.txt

```diff
@@ -1,2 +1,2 @@
 redfish
-pyyaml
+pyyaml>=5.1
```

Comment:
Oops. I bet this is because of my addition of the `Loader=yaml.FullLoader` param in the yaml.load() call. Thanks for catching and fixing.

Reply:
Yup! If people already had 5.1 or higher, it would already print out a nasty warning since "Loader" wasn't specified before.